rpcs3/Emu/Cell/lv2/sys_usbd.cpp

```diff
@@ -387,14 +390,13 @@ void usb_handler_thread::check_devices_vs_ldds()
 					continue;
 				}
 
-				sys_usbd.success("Ldd device matchup for <%s>", ldd.name);
-
 				dev->read_descriptors();
+				dev->assigned_number = dev_counter++; // assign current dev_counter, and atomically increment
+
+				sys_usbd.success("Ldd device matchup for <%s>, assigned as handled_device=0x%x", ldd.name, dev->assigned_number);
```

Comment:
Why is this printed as hex? It's just some counter which is humanly readable as a regular number

Reply:
It seems that sys_usbd.cpp only uses HEX display for everything... like:
L495: ... max_devices=0x%x...
L696: ... buf_length=0x%x....
L699: ...., wIndex:0x%x, wLength:0x%x" ....
L787:  .... count=*0x%x)"....  <-- ok, this one is a little strange

So I didn't want to break that consistency just for my change.
If the plain old decimal format is preferred, I've got no issue with that.

Reply:
whatever you want